### PR TITLE
Copy exec plan test back to lonestar/querying

### PR DESCRIPTION
**Pull request by l-hoang**
The reason is that we need the query plan executable in Lonestar to link
with libraries for distributed execution.

### CODE REVIEW

**Comment by ddn0** [comment]
Can you just copy the file and also give it a better name (like anything that doesn't have test in it)?

Per https://github.com/KatanaGraph/katana/pull/417 we will need infrastructure for running unit tests and separate infrastructure for user facing queries and this is not mutually exclusive.

**Comment by l-hoang** [comment]
> Can you just copy the file and also give it a better name (like anything that doesn't have test in it)?

Works for me.

File: lonestar/querying/cpu/CMakeLists.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 add_subdirectory(querytest)
+add_subdirectory(execute_query_plan)
```
Comment (ddn0) [inline lonestar/querying/cpu/CMakeLists.txt:2]: I thought there was a decision that all the application directories would be named with dashes?

Comment (l-hoang) [reply lonestar/querying/cpu/CMakeLists.txt:2]: yup, let me fix that

File: lonestar/querying/cpu/CMakeLists.txt [hunk 2]
```diff
@@ -1 +1,2 @@
 add_subdirectory(querytest)
+add_subdirectory(execute-query-plan)
```
Comment (roshandathathri) [inline lonestar/querying/cpu/CMakeLists.txt:2]: I think this is misnomer. The input to the executable is a query, not a query plan.

Comment (ddn0) [reply lonestar/querying/cpu/CMakeLists.txt:2]: How about "graph-query"?

**Comment by roshandathathri** [comment]
@l-hoang is this done?

@ddn0 can you please approve if it's done?

I need this to be merged because I'll be modifying ExecutionPlanTest.cpp for #419.

**Comment by l-hoang** [comment]
Since time is limited for Friday I'm actually messing around with querytest at the moment to make it use the op graph rather than porting all the tests over.

There needs to be a decision about what to do with which later though.

**Comment by roshandathathri** [comment]
> Since time is limited for Friday I'm actually messing around with querytest at the moment to make it use the op graph rather than porting all the tests over.

That makes sense. Let's drop this PR. 

In PR #419, I'll drop execution-plan-test (because it is failing). 

In a subsequent PR, change querytest to graph-query and make it work with OpGraph. We can then delete execution-plan-test in libquery or make it so that it is doing unit-test instead of end-to-end test.

**Comment by ddn0** [comment]
> That makes sense. Let's drop this PR.

Works for me.

**Pull request closed without merging**